app/views/govuk_publishing_components/components/docs/list.yml

```diff
@@ -0,0 +1,59 @@
+name: List
+description: A list, unordered or ordered.
+body: |
+  This is an ordered or unordered list, with or without visible bullets / numbers.
+accessibility_criteria: |
+  The list must:
+
+  - inform the user how many items are in the list
+  - convey the content structure
+  - indicate the current page when contents span different pages, and not link to itself
+
+  The contents list may:
```

Comment:
We should be consistent with our terminology, especially since there's already a component named 'Contents list' - this is a little confusing. Is this meant to be here?

Reply:
Yes, sorry - missed that! Fixed now.